### PR TITLE
OBPIH-5317 Export files with product translations (first part)

### PR DESCRIPTION
**Pull request by alannadolny**

### CODE REVIEW

File: grails-app/services/org/pih/warehouse/inventory/InventoryService.groovy [hunk 1]
```diff
@@ -2625,10 +2625,12 @@ class InventoryService implements ApplicationContextAware {
     String exportBaselineQoH(products, quantityMapByDate) {
         def csvrows = []
         NumberFormat numberFormat = NumberFormat.getNumberInstance()
+        def session = RequestContextHolder.currentRequestAttributes().getSession()
```
Comment (kchelstowski) [inline grails-app/services/org/pih/warehouse/inventory/InventoryService.groovy:2628]: what is it added for? I don't see it being used anywhere in this file

File: grails-app/domain/org/pih/warehouse/product/Product.groovy [hunk 2]
```diff
@@ -244,7 +244,8 @@ class Product implements Comparable, Serializable {
                          "applicationTagLib",
                          "handlingIcons",
                          "uoms",
-                         "translatedName"
+                         "translatedName",
+                         "translatedNameWithLocalCode"
```
Comment (kchelstowski) [inline grails-app/domain/org/pih/warehouse/product/Product.groovy:248]: it should be called `Locale` not `Local`

**Review by kchelstowski — CHANGES_REQUESTED** [review]

File: grails-app/domain/org/pih/warehouse/product/Product.groovy [hunk 3]
```diff
@@ -680,6 +681,11 @@ class Product implements Comparable, Serializable {
        return packages.collect { [uom: it.uom.code, quantity: it.quantity] }.unique()
     }
 
+    String getTranslatedNameWithLocalCode() {
+        String localeTag = LocalizationUtil.localizationService.getCurrentLocale().toLanguageTag().toUpperCase()
+        return "${name} ${translatedName ? "(${localeTag}: ${translatedName})" : ''}"
```
Comment (awalkowiak) [inline grails-app/domain/org/pih/warehouse/product/Product.groovy:686]: You could do this as:
`"${name}${translatedName ? " (${localeTag}: ${translatedName})" : ''}"`
(remove space after `${name}` and add it before `(`, to not get dangling space at the and if there is no translation for that product).

File: grails-app/services/org/pih/warehouse/inventory/InventoryService.groovy (hunk 1 above)
Comment (awalkowiak) [reply grails-app/services/org/pih/warehouse/inventory/InventoryService.groovy:2628]: +1

**Review by awalkowiak — CHANGES_REQUESTED** [review]

**Review by kchelstowski — APPROVED** [review]

**Review by awalkowiak — APPROVED** [review]

**Pull request merged**